tensorflow/compiler/mlir/tosa/tests/tfl-to-tosa-pipeline.mlir

```diff
@@ -68,16 +69,16 @@ func @test_conv2d_qi8(%arg0: tensor<1x32x32x8x!quant.uniform<i8:f32, 0.015684768
 
 // -----
 
-// CHECK-LABEL: test_depthwise_conv2d_bias_qi8
-// CHECK-DAG: %[[VAR0:.*]] = "tosa.const"() {value = dense<{{.*}}> : tensor<1x2x2x16xi8>}
```

Comment:
Please change all cases of VAR# -> VAR_# back to VAR# in the fail. The test above is not the only case.

Reply:
I understand you concern but when I run the latest:
bin/tensorflow/compiler/mlir/tf-opt test.mlir --tosa-legalize-tfl | /external/llvm-raw/mlir/utils/generate-test-checks.py

where test.mlir holds the test func, the generated checks are being generated with VAR_. So unless I change them all manually this is going to happen anyway, next time we update the test checks. 

If we think its best, I will update them manually to minimise diffs.

Reply:
> I understand your concern but when I run the latest: bin/tensorflow/compiler/mlir/tf-opt test.mlir --tosa-legalize-tfl | /external/llvm-raw/mlir/utils/generate-test-checks.py
> 
> where test.mlir holds the test func, the generated checks are being generated with VAR_. So unless I change them all manually this is going to happen anyway, next time we update the test checks.
> 
> If we think its best, I will update them manually to minimise diffs.

We should match the previous case. Overall I prefer manually modifying the IR to correctness. Use of automatic tooling to generated the check statements can result in implementation defining tests rather than tests verifying correctness.